src/mui-setup.js

```diff
@@ -19,6 +19,12 @@ export default function muiSetup() {
       boxShadow: '0 0 0 2px #3f8ab3 inset',
       outline: 'none',
     };
+    muiConfig.overrides.MuiToolbar = {
+      regular: {
+        minHeight: '49px !important',
```

Comment:
@haxxmaxx  I do not really want to use` !important`. But I do not understand that our own custom theme is overridden by the default theme.